org.openhab.binding.zigbee/README.md

```diff
@@ -8,7 +8,9 @@ The ZigBee binding supports an interface to a wireless ZigBee home automation ne
 
 A ZigBee Coordinator is the network controller, and is therefore the heart of the ZigBee network. It also acts as the trust centre to control security access to the network.
 
-Coordinators need to be installed manually and the serial port and baud rate must be set. These are set to match the configuration that the dongle is in. Should you wish to use a different baud rate than the default speed of the device, you must change the configuration of the dongle using some other, and then configure the binding to match your change. If in doubt, you should leave the settings at their default values which should work in most cases.
+Coordinators need to be installed manually and the serial port and baud rate must be set. These are set to match the configuration that the dongle is in. Should you wish to use a different baud rate than the default speed of the device, you must change the configuration of the dongle using some other, and then configure the binding to match your change. If in doubt, you should leave the settings at their default values which should work in most cases. 
+
+If you are running on Linux, then you need to add the uesr 'openhab' to the tty group, and enable "Extra_Java_Opts" for the serial port your coordinator uses. Additionally for Docker users, you will need to pass the serial port through Docker to openHAB. You can refer to your installation guide for specific instructions. 
```

Comment:
Small type: `uesr` ->   `user`

Reply:
You mentioned that the installation pages for Linux and Docker mention the necessary steps. What do you think about linking to those installation pages here?

Reply:
I have fixed my typo and have added links to both. Do I need to create another pull request?

Reply:
Great, welcome. I'm just trying to help make a great platform easier to use.

Reply:
@Ted-Corrrell Sorry for late reply :/ Having read your update, and @openhab-5iver 's comment, what do you think about updating the PR as follows?

````
If you are running on Linux, then you probably need to add the user 'openhab' to the tty group, and enable `EXTRA_JAVA_OPTS` for the serial port your coordinator uses (see [Linux install guide (https://www.openhab.org/docs/installation/linux.html#privileges-for-common-peripherals)). Additionally for Docker users, you will need to pass the serial port through Docker to openHAB (see [Docker install guide](https://www.openhab.org/docs/installation/docker.html#explanation-of-arguments-passed-to-docker)).
````

If not, I think one could merge as is (and I would try to add those changes in an extra PR :) ).

Reply:
I didn't see @openhab-5iver 's comment  until just now. having the links point directly to the needed portion of the page is even better than just sending them to the page. I'll make the changes right now.